src/modules/m_override.cpp

```diff
@@ -199,6 +201,46 @@ class ModuleOverride : public Module
 		return MOD_RES_PASSTHRU;
 	}
 
+	// Begin override for CHANMSG (bypassing privmsg restirctions such as muteban and chanfilter)
+	ModResult OnUserPreMessage(User* user, void* dest, int target_type, std::string& text, char status, CUList& exempt_list, MessageType msgtype) CXX11_OVERRIDE
+	{
+		if (active)
+			return MOD_RES_PASSTHRU;
+
+		ModResult MOD_RESULT;
+
+		if (target_type == TYPE_CHANNEL && user->IsOper() && CanOverride(user, "CHANMSG"))
+		{
+			active = true;
+			FIRST_MOD_RESULT(OnUserPreMessage, MOD_RESULT, (user, dest, target_type, text, status, exempt_list, msgtype));
+			active = false;
+
+			// If there is something stopping us from sending our privmsg to the channel then we override
+			if (MOD_RESULT == MOD_RES_DENY)
+			{
+				Channel* channel = static_cast<Channel*>(dest);
+				ServerInstance->SNO->WriteGlobalSno('v',user->nick+" used oper override to send a message on "+channel->name);
+				return MOD_RES_ALLOW;
+			}
+			return MOD_RES_ALLOW; // We already ran FIRST_MOD_RESULT, so if there was no MOD_RES_DENy then they were all passthrus, so there's no point in re-running it anyway
```

Comment:
I intentionally kept this line instead of removing it (which would work because of line 225) to clarify the line comment on 225. I can remove this if people want me to.